sdk/storage/storage-file/src/policies/SharedKeyCredentialPolicy.ts

```diff
@@ -76,6 +76,13 @@ export class SharedKeyCredentialPolicy extends CredentialPolicy {
       `SharedKey ${this.factory.accountName}:${signature}`
     );
 
+    // Workaround for https://github.com/axios/axios/issues/2107
+    // We should always keep the 'content-length' header once the issue is solved
+    // For a better explanation about this workaround, look here: https://github.com/Azure/azure-sdk-for-js/pull/3273
+    if (typeof request.body !== "function" && !(request.body && request.onUploadProgress)) {
```

Comment:
We don't **need** it because the two described scenarios are never triggered in storage-queue tests. But I believe we **should** update storage-queue to make sure it behaves correctly. If there's a non-tested situation in which the issue is triggered, then the server will return an error.

Reply:
I have created a PR for storage-queue: https://github.com/Azure/azure-sdk-for-js/pull/3328

Reply:
Nice